pkg/options/driver.go

```diff
@@ -34,3 +37,31 @@ func NewDriverTypes() *DriverTypes {
 		Enum: NewEnum(types, drivertype.TypeKmod),
 	}
 }
+
+// Driver defines options that are common while interacting with driver commands.
+type Driver struct {
+	Type     drivertype.DriverType
+	Name     string
+	Repos    []string
+	Version  string
+	HostRoot string
+}
+
+// ToDriverConfig maps a Driver options to Driver config struct.
+func (d *Driver) ToDriverConfig() *config.Driver {
+	return &config.Driver{
+		Type:     d.Type.String(),
+		Name:     d.Name,
+		Repos:    d.Repos,
+		Version:  d.Version,
+		HostRoot: d.HostRoot,
+	}
+}
+
+// Validate runs all validators steps for Driver options.
+func (d *Driver) Validate() error {
+	if !filepath.IsAbs(d.HostRoot) {
```

Comment:
For now, the only invalid value might be a relative hostroot.
Perhaps we might want to test that eg: all driver repos are indeed urls?

Reply:
Done in latest commit.